client/src/components/report.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Popup from 'reactjs-popup';
+import ReportThankyou from './reportThankyou';
+
+const Report = () => (
+  <Popup trigger={<button>report</button>} position="right center">
+
+    {close => (
```

Comment:
What is close?

Reply:
That gives the little x in the top left corner that you can press to exit the popup.